llvm/docs/LangRef.rst

```diff
@@ -5418,6 +5418,8 @@ X86:
   operand will get allocated only to RAX -- if two 32-bit operands are needed,
   you're better off splitting it yourself, before passing it to the asm
   statement.
+- ``jR``: An 8, 16, 32, or 64-bit integer EGPR when EGPR feature is on.
+  Otherwise, same as ``R``.
```

Comment:
I think it's same as `r` instead of `R` if egpr is off. Could I know why we design it as  `jR` instead of `jr`?

Reply:
`r` is same as `R` for X86 IIUC. jR is to follow gcc

Reply:
I think no
```
- ``r`` or ``l``: An 8, 16, 32, or 64-bit integer register.
- ``R``: An 8, 16, 32, or 64-bit "legacy" integer register -- one which has
  existed since i386, and can be accessed without the REX prefix.
```

`jR` seems weird by the existing conventions. We can change the design before we have the real users.

Reply:
[bf3a53c](https://github.com/llvm/llvm-project/pull/92338/commits/bf3a53c0b7fc1828572f771e4772d25062110dc0) extended jr, l, q